Color.js

```diff
@@ -1,12 +1,12 @@
 class Color {
   constructor() {
-    this.id = Math.floor(Math.random() * 10000).toString();
-    this.hexCode = this.makeRandomColor();
-    this.isLocked = false;
+    this.id = Math.floor(Math.random() * 10000).toString()
+    this.hexCode = this.makeRandomColor()
+    this.isLocked = false
   }
 
   makeRandomColor() {
-    var hexCode = "#";
```

Comment:
Removed semi-colons to be consistent with previous formatting

Reply:
Looks great. Thanks for cleaning that up.